data/cpu_bugs/autoyast/SLE-12-SP4/autoyast_gnome.xml

```diff
@@ -0,0 +1,161 @@
+      <rate/>
+    </keyboard_values>
+    <keymap>english-us</keymap>
+  </keyboard>
+  <language>
+    <language>en_US</language>
+    <languages/>
+  </language>
+  <login_settings/>
+  <report>
+    <errors>
+      <log config:type="boolean">true</log>
+      <show config:type="boolean">true</show>
+      <timeout config:type="integer">0</timeout>
+    </errors>
+    <messages>
+      <log config:type="boolean">true</log>
+      <show config:type="boolean">true</show>
+      <timeout config:type="integer">10</timeout>
+    </messages>
+    <warnings>
+      <log config:type="boolean">true</log>
+      <show config:type="boolean">true</show>
+      <timeout config:type="integer">10</timeout>
+    </warnings>
+    <yesno_messages>
+      <log config:type="boolean">true</log>
+      <show config:type="boolean">true</show>
+      <timeout config:type="integer">10</timeout>
+    </yesno_messages>
+  </report>
+  <services-manager>
+    <default_target>multi-user</default_target>
+    <services>
+      <disable config:type="list"/>
+      <enable config:type="list">
+        <service>sshd</service>
+      </enable>
+    </services>
+  </services-manager>
+  <software>
+    <image/>
+    <instsource/>
+    <patterns config:type="list">
+      <pattern>Minimal</pattern>
+      <pattern>32bit</pattern>
+      <pattern>base</pattern>
+    </patterns>
+    <packages config:type="list">
+      <package>autoyast2</package>
+      <package>yast2</package>
+      <package>vim</package>
+      <package>iputils</package>
+      <package>less</package>
+      <package>screen</package>
+      <package>lsof</package>
+      <package>pciutils</package>
+      <package>openssh</package>
+      <package>grub2</package>
+    </packages>
+  </software>
+  <timezone>
+    <hwclock>CST</hwclock>
+    <timezone>Asia/Beijing</timezone>
+  </timezone>
+  <user_defaults>
+    <expire/>
+    <group>500</group>
+    <groups/>
+    <home>/home</home>
+    <inactive/>
+    <no_groups config:type="boolean">true</no_groups>
+    <shell/>
+    <skel/>
+    <umask>022</umask>
+  </user_defaults>
+  <users config:type="list">
+    <user>
+      <encrypted config:type="boolean">false</encrypted>
+      <fullname>Bernhard M. Wiedemann</fullname>
+      <gid>100</gid>
+      <home>/home/bernhard</home>
+      <password_settings>
+        <expire/>
+        <flag/>
+        <inact>-1</inact>
+        <max>99999</max>
+        <min>0</min>
+        <warn>7</warn>
+      </password_settings>
+      <shell>/bin/bash</shell>
+      <uid>1000</uid>
+      <user_password>nots3cr3t</user_password>
+      <username>bernhard</username>
+    </user>
+    <user>
+      <encrypted config:type="boolean">false</encrypted>
+      <fullname>root</fullname>
+      <gid>0</gid>
+      <home>/root</home>
+      <password_settings>
+        <expire></expire>
+        <flag></flag>
+        <inact></inact>
+        <max></max>
+        <min></min>
+        <warn></warn>
+      </password_settings>
```

Comment:
looks like some sections are superfluous. Do you actually need them or would it be ok to leave these sections and potentially others out?

Reply:
actually, I just copy them from others and add something that what I need, I never think about remove something about it. :-). I could remove them.